tests/bigquery/pipeline_tests.py

```diff
@@ -116,57 +114,114 @@ def test_get_extract_parameters(self):
     }
 
     self.assertDictEqual(actual_extract_config, expected_extract_config)
-    self.assertDictEqual(execute_config, {'table': 'test_table'})
+
+  @mock.patch('google.datalab.utils.commands.get_notebook_item')
+  def test_get_execute_parameters(self, mock_notebook_item):
+    mock_notebook_item.return_value = google.datalab.bigquery.Query(
+        'SELECT @column FROM publicdata.samples.wikipedia where endpoint=@endpoint')
+
+    transformation_config = {
+      'query': 'foo_query'
+    }
+    output_config = {
+      'table': 'foo_table',
+      'mode': 'foo_mode'
+    }
+    parameters_config = [
```

Comment:
Can we simplify these? Or in another change?

Reply:
Different change. I have a bug on it already: https://github.com/googledatalab/pydatalab/issues/540